File: programing/C++ programs/Practice OOP C ++/Average.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    float a[10],sum=0.0;
    cout<<"Enter Number of elements(1 to 10): ";
    cin>>n;
    while(n>10 || n<0){
        cout<<"Wrong input!!!\n";
        cout<<"Enter Number of elements(1 to 10): ";
        cin>>n;
    }
    for(int i=0; i<n; i++){\
        cout<<"Enter numbers "<<i+1<<" : ";
        cin>>a[i];
        sum += a[i];
    }
    cout<<sum/n;
}

File: programing/Java programs/Practice/Average.java
import java.util.Scanner;
public class Average {
    public static void main(String []args){
        float sum = 0;
        float a[] = new float[10];
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter no of elements: ");
        float x = sc.nextFloat();     
        for(int i=0; i<x; i++){
            System.out.print("Enter "+ i+1 + " numbers : ");
            a[i] = sc.nextFloat();
        }
        for(int i=0; i<x; i++){
            sum += a[i]; 
        }
        float average = sum/x;
        System.out.format("Average  is: %.2f",average);
    }
}
